ecc/enclave/enclave_stub.go

```diff
@@ -367,50 +368,65 @@ func (e *StubImpl) Invoke(args []byte, pk []byte, shimStub shim.ChaincodeStubInt
 	signaturePtr := C.malloc(SIGNATURE_SIZE)
 	defer C.free(signaturePtr)
 
+	// get signed proposal
+	signedProposal, err := shimStub.GetSignedProposal()
+	if err != nil {
+		return nil, fmt.Errorf("cannot get signed proposal")
+	}
+	signedProposalBytes, err := proto.Marshal(signedProposal)
+	if err != nil {
+		return nil, fmt.Errorf("cannot get signed proposal bytes")
+	}
+	signedProposalPtr := C.CBytes(signedProposalBytes)
+	defer C.free(unsafe.Pointer(signedProposalPtr))
+
+	//ASSUME HERE b64ChaincodeRequestMessagePtr is not the protobuf, so let's buildit (rmeove block later)
+	crmProto := &fpcpb.ChaincodeRequestMessage{
+		EncryptedRequest: args,
+	}
+	crmProtoBytes, err := proto.Marshal(crmProto)
+	if err != nil {
+		return nil, fmt.Errorf("marshal error")
+	}
+	b64crm := base64.StdEncoding.EncodeToString(crmProtoBytes)
```

Comment:
`initEnclave` we seem to do just serialized proto with no base64: Let's do it consistent for EDL interface, either all base64 or no base64. Seems easiest just to send protobuf, contrary to ecc<->client interface there is no need for ascii as the peer shell wrapper requires?  (of course it would then eventually require a base64-decode of that message once marcus merges but i think he does that already)

Reply:
right, encoding removed